client/src/components/profile/Socials.tsx

```diff
@@ -34,7 +34,7 @@ const Socials = ({ profileData }: UserProps): ReactElement => {
                 target='_blank'
                 title='LinkedIn Profile'
               >
-                <LinkedInIcon fontSize='large' />
+                <LinkedInIcon fontSize='large' sx={{ color: 'blue' }} />
```

Comment:
Is this "LinkedIn blue" ? Might want to pick the color straight from the linkedIn logo so it's an exact match

Reply:
Added the LinkedIn Hex Code for their color